### PR TITLE
Update RxSwift dependency to 4.0.0

### PR DESCRIPTION
**Pull request by devxoul**
Sets RxSwift dependency to `~> 4.0`.

### CODE REVIEW

File: Moya.podspec [hunk 1]
```diff
@@ -36,6 +36,6 @@ Pod::Spec.new do |s|
   s.subspec "RxSwift" do |ss|
     ss.source_files = "Sources/RxMoya/"
     ss.dependency "Moya/Core"
-    ss.dependency "RxSwift", "4.0.0-rc.0"
+    ss.dependency "RxSwift", ">= 4.0.0"
```
Comment (SD10) [inline Moya.podspec]: Do we want `>=` or just `4.0.0`?

Comment (BasThomas) [reply Moya.podspec]: I would say `~> 4.0` for both CocoaPods and Carthage, like we are dong with the other dependencies.

Comment (devxoul) [reply Moya.podspec]: Ohh that's right. It is not a beta anymore. I just updated to use `~>` :)

**Comment by sunshinejr** [comment]
@devxoul Thanks! We should also update `Package.swift` I believe.

**Comment by devxoul** [comment]
@sunshinejr sure! Did it :)

**Pull request merged**